package.json

```diff
@@ -78,7 +78,9 @@
     "CHANGELOG.md",
     "LICENSE",
     "CONTRIBUTING.md",
-    "README.md"
+    "README.md",
+    "src/*",
```

Comment:
do we want this?

Reply:
inline-source-map works so that we need the src/* and types/* folder in order to display the typescript file